pkg/action/install.go

```diff
@@ -631,7 +631,7 @@ func createOrOpenFile(filename string, append bool) (*os.File, error) {
 	return os.Create(filename)
 }
 
-// check if the directory exists to create file. creates if don't exists
+// check if the directory exists to create file. creates if don't exist
```

Comment:
```suggestion
// check if the directory exists to create file. creates if does't exist
```

Reply:
I've thrown in a few grammar corrections including this one 👍